app/src/main/java/mod/hilal/saif/activities/tools/ConfigActivity.java

```diff
@@ -53,11 +79,25 @@ public class ConfigActivity extends Activity {
     public static final String SETTING_SKIP_MAJOR_CHANGES_REMINDER = "skip-major-changes-reminder";
     public static final String SETTING_BLOCKMANAGER_DIRECTORY_PALETTE_FILE_PATH = "palletteDir";
     public static final String SETTING_BLOCKMANAGER_DIRECTORY_BLOCK_FILE_PATH = "blockDir";
+    public static final String SETTING_APP_THEME = "sketchware-theme";
```

Comment:
This commit only mentions "redesign" while it also adds a new setting. That is not OK. The commit is definitely not a refactor btw, refactors don't change the behavior or looks of the app.

Reply:
Bro, as I said on discord, I'm making a theme system right?, and I took this code from the repository where I'm doing this, and I tried to remove the lines related to the theme, this line went unnoticed by me